Library/Homebrew/tap.rb

```diff
@@ -673,8 +673,13 @@ def new_cask_path(token)
 
   sig { params(token: String).returns(String) }
   def relative_cask_path(token)
-    new_cask_path(token).to_s
-                        .delete_prefix("#{path}/")
+    if token.start_with?("font-")
+      new_cask_font_path(token).to_s
+                               .delete_prefix("#{path}/")
+    else
+      new_cask_path(token).to_s
+                          .delete_prefix("#{path}/")
+    end
```

Comment:
```suggestion
    if token.start_with?("font-")
      new_cask_font_path(token)
    else
      new_cask_path(token)
    end.to_s.delete_prefix("#{path}/")
```

Reply:
This is what I had in mind before discovering the other issue. I'll push this through now.